Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: *
+
+all: build-base build-env 
+
+build-base:
+	git clone --depth 1 https://github.com/sunil-sahoo/opencart.git -b docker-deploy /project
```

Comment:
You shouldn't have it in this file. Makefile must be only for building and setting up stuff before running compose